### PR TITLE
[table] CSS refactor

### PR DESCRIPTION
**Pull request by tmorehouse**
When `b-table` had prop `small`, the arrows were not aligned top to bottom correctly (up to high).

Also removed erroneous `.` from `table-responsive` class

### CODE REVIEW

File: lib/components/table.vue [hunk 1]
```diff
@@ -318,6 +320,29 @@
         opacity: 0.3;
     }
 
+    table.table-sm thead .sorting:before,
```
Comment (pi0) [inline lib/components/table.vue]: Can't we refactor it into a generalized prop selector? This is a LOT! :D

Comment (tmorehouse) [inline lib/components/table.vue]: Yeah, I was thinking the same.

Maybe making asc/desc their own class, and maybe making disabled its own class.

Comment (tmorehouse) [inline lib/components/table.vue]: so have the following classes:
- `sorting`
- `asc`
- `desc`
- `disabled`

Comment (tmorehouse) [inline lib/components/table.vue]: I can try and refactor before the merge.

Comment (tmorehouse) [inline lib/components/table.vue]: Also not sure why pagination is being imported, as it is not used directly by table.

**Comment by tmorehouse** [comment]
CSS has been refactored for much smaller footprint

**Comment by pi0** [comment]
Nice +1

**Pull request merged**